react/fields/MonthPicker/MonthPicker.js

```diff
@@ -0,0 +1,175 @@
+import styles from './MonthPicker.less';
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+
+import ErrorIcon from '../../icons/ErrorIcon/ErrorIcon';
+import CustomMonthPicker from './CustomMonthPicker/CustomMonthPicker';
+import NativeMonthPicker from './NativeMonthPicker/NativeMonthPicker';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class MonthPicker extends Component {
+
+  static displayName = 'MonthPicker';
+
+  static propTypes = {
+    /* eslint-disable consistent-return */
+    id: (props, propName, componentName) => {
```

Comment:
You're right. There are a lot of duplicates in style guide atm. Also we want to pull label and message (and any other common patterns) into reusable components so we're not duplicating so much. Feel both of these would be better off having a dedicated PR though.